doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -191,6 +191,41 @@ name "/Q30/"
 
 \begin_layout Labeling
 \labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/Q40/"
+
+\end_inset
+
+ Beagle needs a define time to run in dependencey of the 
```

Comment:
How is this impossible to fulfil. Lets say we have a matrix like this:
`50    52    49    54`
`57    59    X`

where `X` is the value to be measured next. It can be anything, but we know that the ones around it took between 49 and 59 units of time. So we might just set the timeout to fifteen times that and see if it terminates within in the given period of time. If it does: Great, we got another value. If it doesn't: Oops, this is probably going to take either a long time or we got ourselves an infinity loop here. Maybe come back later when we found a pattern describing this or if we don't, leave it out.